network/netplay/netplay.c

```diff
@@ -549,6 +550,23 @@ static bool netplay_get_cmd(netplay_t *netplay)
          {
             uint32_t frame;
 
+            /* Make sure we're ready for it */
+            if (netplay->quirks & NETPLAY_QUIRK_INITIALIZATION)
+            {
+               if (!netplay->is_replay)
+               {
+                  netplay->is_replay = true;
+                  netplay->replay_ptr = netplay->self_ptr;
+                  netplay->replay_frame_count = netplay->self_frame_count;
+                  netplay_wait_and_init_serialization(netplay);
+                  netplay->is_replay = false;
+               }
+               else
+               {
+                  netplay_wait_and_init_serialization(netplay);
+               }
```

Comment:
Oh, you mean because it switches replay differently? Yeah, either way we have to wait and initialize to load a state, but we can only "wait" in replay mode, so... yuck. Not sure what else to do about it tho.